crates/dojo-core/src/world.cairo

```diff
@@ -394,7 +393,8 @@ mod world {
         /// * `keys` - The keys of the event.
         /// * `values` - The data to be logged by the event.
         fn emit(self: @ContractState, mut keys: Array<felt252>, values: Span<felt252>) {
-            self.caller_system().serialize(ref keys);
+            let system = _system_name_from_caller(self);
```

Comment:
let's not use the `_` prefix for internals. it's a bit of an anti pattern in cairo / rust since `_` prefix usually signifies unused variables

Reply:
# 🫡